ui/perfherder/tests/TestsTable.jsx

```diff
@@ -101,11 +101,11 @@ export default function TestsTable(props) {
   ];
 
   return (
-    <Container fluid className="my-3 px-0">
+    <Container fluid className="my-3 px-0 vh-100">
       <ReactTable
         data={results}
         columns={columns}
-        className="-striped -highlight mb-5"
+        className="-striped -highlight mb-5 h-100"
```

Comment:
If we add tests-table class to the table, you can then target the rows within the table and lower their hight to 35 px, forcing them to not stretch all over the table height.

![image](https://user-images.githubusercontent.com/88494730/153830297-62aa28a0-3645-4681-ae3c-5dfde984d020.png)